<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainAppWindow</class>
 <widget class="QMainWindow" name="MainAppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="serviceSearchLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QListWidget" name="searchResultsListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="bookingDateEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата бронирование</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="bookingTimeEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Время бронирование</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="peopleCountLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество человек</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Примечания (по желанию)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bookButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Забронировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Список бронирований</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBookingsButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Мои бронирования</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="bookingsTableWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>311</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelBookingButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editBookingButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPeopleCountLineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="newBookingDateEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="newBookingTimeEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="viewHistoryButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Просмотр истории</string>
    </property>
   </widget>
   <widget class="QListWidget" name="historyListWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
